jerry-core/lit/lit-globals.h

```diff
@@ -94,6 +94,16 @@ typedef ecma_char_t *ecma_char_ptr_t;
 #define LIT_UTF8_MAX_BYTES_IN_CODE_POINT (4)
```

Comment:
Could you, please, clarify, do we need the define after switching to CESU-8?

Is it correct that CESU-8 uses UTF-8 encoding scheme, but only for 1-byte and 2-byte code points?
Shouldn't we rename all `UTF8` / `utf8` definitions to `CESU8` / `cesu8`?